res/xml/bengali_provat.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<keyboard>
+	<row>
+		<key 				key0="দ"	key1="ধ"	key2="১"	key3="!"				key5="esc"	/>
```

Comment:
Symbols close to the edges of the keyboard can be hard to reach. What do you think of moving `esc` to the Shift key below and `ধ` to `key4` ?

Reply:
I know, but I think it's better to keep identical as physical layout.

@bdeshi has a suggestion to fix this issue. At the moment, shift key is only working for latin alphbets. To make shift key work in onthet languages, something like this can be a good idea:
# layout definition
...
<key key0="দ" shifted0="ধ"/>
then update the apply_shift() function so that it reads the corresponding shifted# character for pressed key# from the layout file.

Reply:
This is an interesting idea but it might make layouts more complex. The current way would be to add new rules into `map_char_shift`, like this:
```
  case 'দ': return 'ধ';
```

Would that work for you ?

Reply:
If shift is applied to a specific language instead of a language, then it won't work. Different layouts map keys differently; for example, in Provat, 'ম' is mapped with 'ঙ' but in Bijoy, 'ম' is mapped with 'শ'.

Reply:
Interesting!

The `shifted0` is not bad but hard to implement currently. I'll try to come up with a solution soon.

Reply:
Thanks!

Reply:
I finally found some time and implemented something that might help (22d407c):
```xml
<keyboard>
  <modmap>
    <shift a="ম" b="ঙ"/>
  </modmap>

  <!-- The <row>s here -->
</keyboard>
```

This is brand new and might contains bugs.